<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvd::LayerStackWidget</class>
 <widget class="QWidget" name="mvd::LayerStackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::CopyAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="topButton">
       <property name="toolTip">
        <string>Move layer to top of stack.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerTop_Icon</normaloff>:/icons/action_LayerTop_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="upButton">
       <property name="toolTip">
        <string>Move layer up in to stack.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerUp_Icon</normaloff>:/icons/action_LayerUp_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="downButton">
       <property name="toolTip">
        <string>Move layer down in the stack.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerDown_Icon</normaloff>:/icons/action_LayerDown_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bottomButton">
       <property name="toolTip">
        <string>Move layer to bottom of stack.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerBottom_Icon</normaloff>:/icons/action_LayerBottom_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="projectionButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerProjection_Icon</normaloff>:/icons/action_LayerProjection_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_ApplyAll_Icon</normaloff>:/icons/action_ApplyAll_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadButton">
       <property name="toolTip">
        <string>Reload data file.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_Reload_Icon</normaloff>:/icons/action_Reload_Icon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Application/Monteverdi2/mvdMainWindow.qrc">
         <normaloff>:/icons/action_LayerDelete_Icon</normaloff>:/icons/action_LayerDelete_Icon</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Application/Monteverdi2/mvdMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
